### DISCUSSION TITLE
Should we deprecate the observability chart?

### DISCUSSION THREAD

**Discussion by psschwei** [Qiskit/qiskit-serverless#997] [Ideas]
To my knowledge, the [observability chart](https://github.com/Qiskit-Extensions/quantum-serverless/tree/main/charts/qs-observability) isn't being used. Should we consider deprecating it?

**Comment by Tansito** [comment]
It's a good conversation. For me it's a similar situation to terraform. We are not using it directly but maybe for external users is useful. My position here is like in terraform, taking to account the amount of work that we have and the number of people that we are... I think we could remove it to simplify configuration and maintenance.

**Comment by akihikokuroda** [comment]
We may replace the chart with some docs that explain some examples of the observability tools and their capability and how to install and configure them.
